vite.config.js

```diff
@@ -11,15 +11,14 @@ const appName = JSON.parse(packageJson).appName
 const appVersion = JSON.parse(packageJson).version
 const sbcName = JSON.parse(packageJson).sbcName
 const sbcVersion = JSON.parse(packageJson).dependencies['sbc-common-components']
-const aboutApp = (appName && appVersion) ? `${appName} v${appVersion}` : 'Unknown APP'
-const aboutSbc = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : 'Unknown SBC'
```

Comment:
If we can't parse the properties from the package.json file then we have bigger problems, so I simplified this.